MANIFEST.in

```
@@ -1 +0,0 @@
-prune xarray/datatree_*
```

I wonder if we should prune everything except `xarray/datatree_/datatree`? Nobody should type-check our docs (we don't either, but that's a different issue) and any other non-python files.

```suggestion
prune xarray/datatree_*
recursive-include xarray/datatree_/datatree*
```

---

So rather than add back all of the datatree_ directory, you're suggesting just the code. That makes a lot of sense to me. 👍